prbt_hardware_support/test/unittests/unittest_libmodbus_client.cpp

```diff
@@ -282,5 +284,16 @@ TEST(LibModbusClientTests, setResponseTimeout)
 int main(int argc, char *argv[])
 {
   testing::InitGoogleTest(&argc, argv);
+
+  //Set logger level for server mock
+  if( ros::console::set_logger_level("ros.prbt_hardware_support.ServerMock", ros::console::levels::Debug) ) {
```

Comment:
please dont set logger levels programmatically

Reply:
Really would like a better solution but for now and for tests this should be fine. As long as there is now way using environment variables with rosconsole...

Reply:
Suggestion?

Reply:
Ok so in case someone wants to set the loggerlevel for Travis/Industrial_CI see
https://gist.github.com/agutenkunst/2f96826b96696784eefb985f73222a58

Reply:
Removed the part

Reply:
What you did